java/src/jmri/jmrix/JmrixBundle.properties

```diff
@@ -101,6 +101,8 @@ ConnectionPrefixDialog  = Connection Prefix {0} is already assigned
 ConnectionNameDialog    = Connection Name {0} is already assigned
 TipBaudRateFixed        = The baud rate is fixed for this protocol
 TipBaudRateMatch        = Must match the baud rate setting of your hardware
+OutputIntervalLabel     = Output Interval (Msec):
```

Comment:
Did the change to this label (to use `ms`) get missed?

Reply:
Hesitated to disturb the test/review cycle underway. Now added just this one change to jmrix.proporties in commit https://github.com/JMRI/JMRI/pull/6177/commits/8a845a1673ed6b746a974d9027fddff1d97dfeb5
Screen shot updated to confirm.